content/blog/2021-09-30-september-21-community-gems.md

```diff
@@ -0,0 +1,127 @@
+[Data Registry use case](https://dvc.org/doc/use-cases/data-registries) where
+you can learn more about data registry management and how other projects can
+interact with the registry.
+
+### [Is there a way to share data across multiple on-premise machines so that users can train models individually?](https://discord.com/channels/485586884165107732/563406153334128681/879718738163826698)
+
+This is a good scenario to try out
+[sharing a DVC cache](https://dvc.org/doc/user-guide/how-to/share-a-dvc-cache).
+You can have a single storage location mounted on each workstation to serve as a
+central cache.
+
+That way all of your machine learning engineers can work with the same data
+without needing to duplicate it on each machine.
+
+Thanks for the question @fchpriani!
+
+### [If we change the remote we are using in our workspace, does that effect where DVC pulls and pushes data to for all historical commits?](https://discord.com/channels/485586884165107732/563406153334128681/882951655979622400)
+
+Thanks for bringing this up @mattlbeck!
+
+Right now DVC just uses whichever remote is configured in a respective commit
+that you've checked out.
+
+To clarify things a bit more, if you run `dvc push/pull` in a workspace with a
+new remote, that new remote will be used for `--all-branches`, `--all-tags`, and
+`--all-commits`.
+
+### [Is there a command to execute only a few specific stages in a DVC pipeline?](https://discord.com/channels/485586884165107732/485596304961962003/888054401640562698)
+
+You can freeze the stages that you do not want to be executed.
+
+`dvc freeze` and `dvc unfreeze` help you do this. Or you can use
+`dvc repro --glob pattern*` together with `-s` to match the stages you want to
+run.
+
+Thanks for the question @LucZ!
+
+### [When running queued experiments, is it expected for DVC to run `dvc checkout` for each experiment?](https://discord.com/channels/485586884165107732/485596304961962003/883144885417431081)
+
+This brings up a good point, so thanks @dmh!
+
+If you usually run experiments with `dvc repro`, you'll notice that it doesn't
+checkout any files. That's because the experiment is running in the current
+workspace.
+
+When you use `dvc exp run --queue` or `dvc exp run --run-all`, it runs each
+experiment in its own separate temp workspace, so files have to be checked out
+into those workspaces. Check out the notes in
+[this reference doc on queueing and parallel execution](https://dvc.org/doc/command-reference/exp/run#queueing-and-parallel-execution)
+for more details.
+
+### [When working with a data registry, is it possible to pull a specific project folder, modify it, then push Git changes and `dvc push` to the remote storage without pulling data from all the directories?](https://discord.com/channels/485586884165107732/485596304961962003/887427010044002345)
+
+This is definitely possible. A common way to handle this use case, but you have
+another option. You can `dvc pull -R` from the sub-directory, then make your
```

Comment:
Seems like something got mixed up here?

Reply:
Yeah, not sure what happened there, but I cleaned up that sentence.